src/Routes.js

```diff
@@ -15,12 +14,24 @@ import NotFound from "./containers/NotFound";
 export default function Routes({ appProps }) {
   return (
     <Switch>
-      <AppliedRoute path="/" exact component={Home} appProps={appProps} />
-      <UnauthenticatedRoute path="/login" exact component={Login} appProps={appProps} />
-      <UnauthenticatedRoute path="/signup" exact component={Signup} appProps={appProps} />
-      <AuthenticatedRoute path="/settings" exact component={Settings} appProps={appProps} />
-      <AuthenticatedRoute path="/notes/new" exact component={NewNote} appProps={appProps} />
-      <AuthenticatedRoute path="/notes/:id" exact component={Notes} appProps={appProps} />
+      <Route path="/" exact>
+        <Home {...appProps} />
+      </Route>
+      <UnauthenticatedRoute path="/login" exact appProps={appProps} >
```

Comment:
We probably don't need to pass in the `appProps` anymore right?

Reply:
We still need to pass `appProps`  to `UnauthenticatedRoute` and `AuthenticatedRoute` components because they need to render different things based on `appProps.isAuthenticated` value.

Alternatively, we could pass just `isAuthenticated` only. So that it will look like `<UnauthenticatedRoute path="/login" exact isAuthenticated={appProps.isAuthenticated} />`. Let me know if you would like it this way.

Reply:
That's a good point. I missed that!